example_graph_test.go

```diff
@@ -0,0 +1,167 @@
+package redisgraph_test
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+	"github.com/RedisGraph/redisgraph-go"
+	"github.com/gomodule/redigo/redis"
+	"io/ioutil"
+	"log"
+	"os"
+)
+
+func ExampleGraphNew() {
+	conn, _ := redis.Dial("tcp", "0.0.0.0:6379")
+
+	graph := redisgraph.GraphNew("social", conn)
+
+	q := "CREATE (w:WorkPlace {name:'RedisLabs'})"
+	res, _ := graph.Query(q)
+
+	q = "MATCH (w:WorkPlace) RETURN w"
+	res, _ = graph.Query(q)
+
+	res.Next()
+	r := res.Record()
+	w := r.GetByIndex(0).(*redisgraph.Node)
+	fmt.Println(w.Label)
+	// Output: WorkPlace
+}
+
+func ExampleGraphNew_pool() {
+	host := "localhost:6379"
+	password := ""
+	pool := &redis.Pool{Dial: func() (redis.Conn, error) {
+		return redis.Dial("tcp", host, redis.DialPassword(password))
```

Comment:
Why not keep it simple and omit `password` ?

Reply:
solved per new commit